backend/graminsta/post/models.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: UTF-8 -*-
+"""models.py"""
+from django.db import models
+from django.contrib.auth import models as auth_models
+
+
+class Post(models.Model):
+    """
+    A Post is usually user-generated which contains information
+    like an image, and some description to the image.
+    """
+    publisher = models.ForeignKey(auth_models.User, on_delete=models.CASCADE,
+                                  related_name="published_posts")
+    description = models.TextField()
+    img = models.ImageField(upload_to='img')
+    marked_user = models.ManyToManyField(auth_models.User,
+                                         related_name="marking_users")
+    time = models.DateTimeField(auto_now=True)
```

Comment:
What's this timestamp? When the post is created? If so, I'd suggest to name it as `created_at`, and `auto_now_add=True` instead of `auto_now=True`, as the latter will update this field every time you save the object , it's useful for saving the `updated_at` timestamp.